includes/admin/post-meta.php

```diff
@@ -434,11 +434,19 @@ function markup_published( $status_meta ) {
  * @return string
  */
 function markup_error( $status_meta ) {
+	$learn_more = '';
+	if ( 'When authenticating requests to the Twitter API v2 endpoints, you must use keys and tokens from a Twitter developer App that is attached to a Project. You can create a project via the developer portal.' === $status_meta['message'] ) {
```

Comment:
@iamdharmesh, is this message coming from Twitter? Do you know if Twitter responds in English always?
I want to ensure that it succeeds Twitter below to a different language.

Reply:
Yes @ravinderk, this message coming from Twitter. I tried to check but didn't found anywhere that twitter respond error message other than English language.